packages/client/utils/AtlassianManager.ts

```diff
@@ -307,6 +325,15 @@ export default abstract class AtlassianManager {
       return res
     }
     const json = (await res.json()) as AtlassianError | JiraNoAccessError | JiraGetError | T
+
+    if (res.status === 429) {
```

Comment:
ah good point, this was an attempt to resolve your prior suggestion:

> message should probably be the error that jira gives us instead of error because they might give something more informative so we can use that to debug when that error is hit

afaik atlassian doesn't give specific details on the shape. i'll revert back to the hard coded message for now, but did you have a different idea about where the error msg would come from? @mattkrick

Reply:
ah good point, this was an attempt to resolve your prior suggestion:

> message should probably be the error that jira gives us instead of error because they might give something more informative so we can use that to debug when that error is hit

afaik atlassian doesn't give specific details on the shape. i'll revert back to the hard coded message for now, but did you have a different idea about where the error msg would come from? @mattkrick